src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -606,11 +605,9 @@ void DistributedSearchTreeImpl<DeviceType>::communicateResultsBack(
       Kokkos::ViewAllocateWithoutInitializing(ids.label()), n_imports);
 
   sendAcrossNetwork(distributor, export_indices, import_indices);
-  sendAcrossNetwork(distributor, export_ranks, import_ranks);
```

Comment:
But then there is also no point in declaring `export_ranks` and `import_ranks` at all.

Reply:
Sorry, you are right. Was just winging it. Will force push with them removed.